apps/dot_ext/management/commands/authflow_id_delete_and_vacuume.py

```diff
@@ -0,0 +1,67 @@
+import time
+from django.db import connection
+
+from datetime import date, timedelta
+from django.core.management.base import BaseCommand
+
+from apps.dot_ext.models import AuthFlowUuidCopy
+
+DELETE_WITH_LIMIT = "DELETE FROM {table_name} WHERE auth_uuid IN (SELECT auth_uuid FROM {table_name} WHERE created < '{age_date}'::date ORDER BY created LIMIT {limit_on_delete})"
+VACUUM_TABLE = "VACUUM FULL {table_name}"

```

Comment:
Was just looking through the docs for this... I think we want to be pretty careful about this: `requires an exclusive lock on each table while it is being processed`. Sounds like we basically wont be able to interact with this table while the vaccuum is being performed and the FULL is slow according to the docs. I don't know if we should be doing this. We'd definitely need to do some perf testing to determine the impact.